Main/Source/char.cpp

```diff
@@ -8738,6 +8769,12 @@ void character::LeprosyHandler()
   CheckDeath(CONST_S("killed by leprosy"));
 }
 
+void character::VampirismHandler()
+{
+  EditExperience(WISDOM, -25, 1 << 1);
```

Comment:
I copied this behaviour and the numbers from leprosy. I think vampirism might be lame, so I don't want to deter people too much from attempting it as a strategy. If vampirism ends up being overpowered then the frequency of wisdom penalty could be increased.